application_form/target.md

```diff
@@ -0,0 +1,57 @@
+
+Many trans and non-binary people encounter difficulties in life, and in entering technology, that are not encountered by cisgender people - especially compared to cis men. Therefore, it can make sense to open your workshop to this wider group.
+
+## Being open to women
+
+Opening up applications to women, including trans women, is a common choice for Django Girls events. The website defaults to this, but we recommend adding an explicit mention of being trans inclusive. A good way to word this is:
+
+> We are trans-inclusive and welcome applications from all women.
+
+## Being open to women and non-binary people
+
+Another choice is to be open to women and non-binary people. Non-binary people can be AMAB or AFAB, and can have any range of appearances. Some might present in a way that others will read as them being men or women, some look more ambiguous. This is influenced by personal preference, but also by safety concerns. Therefore, you need to be aware that some attendees may identify as non-binary, yet not look different to you than cis men or cis women.
+
+The Django Girls name and website refer to "women" and "girls" in a few places already, so if you use the phrase "women and non-binary people" on your event page, people are likely to assume that you mean "non-binary people who look like women to the organisers" or "people who are feminine enough in their appearance" or "only AFAB non-binary people". To avoid this, it's important to be very explicit.
+
+A good start for being and appearing inclusive is:
+
+- Early on in your event page, clarify your target group as: “women and non-binary people (regardless of gender expression or assigned gender at birth)”
+- Consistently using “women and non-binary people” everywhere, and never “women” alone.
+- Ask for people’s pronouns in the application form - this target group means you can’t assume "she" as the default pronoun. If you make name badges, try to print the pronouns on them.
+
+## Being open to people of any gender
+
+You can also choose to open your event to any applicants, regardless of their gender. However, when you organise an event under the Django Girls name, you must prioritise people who are not cisgender men.
+
+To meet this requirement, you will have to ask for the gender of applicants in your application form. Do not restrict this choice to man/woman, as non-binary people generally do not fit in these categories. A good set of choices can be:
+
+- Man
+- Woman
+- Non-binary/other gender
+
+This allows you to assume that people who answer "man" are cisgender men, and allows you to prioritise other applications over these.
```

Comment:
Is that true? If presented with these options, I’d expect at least some trans men to pick the answer “Man”.

Reply:
Yes, entirely right. I've clarified this a bit. The issue is that if you include trans men in your target group, you'd have to ask for "men" and "trans men" separately, but usually I'd suggest not adding separate options for trans if not needed - trans women are women, after all.